packages/@okta/vuepress-theme-prose/components/FrontPageCodeMirror.vue

```diff
@@ -9,7 +9,7 @@
 
 <script>
 import "codemirror/lib/codemirror.css";
-import "codemirror/theme/material.css";
+// import "components/codemirror.scss",
```

Comment:
Is it possible to use the locally scoped stylesheet here rather than importing it inside of the top level globally?

Reply:
It uses mixins and variables, I am getting errors when trying to load it locally scoped